script.js

```diff
@@ -12,42 +12,33 @@ const country_list = ["Afghanistan","Albania","Algeria","Andorra","Angola","Angu
 	,"Turkey","Turkmenistan","Turks and Caicos","Uganda","Ukraine","United Arab Emirates","United Kingdom","United States","United States Minor Outlying Islands","Uruguay"
   ,"Uzbekistan","Venezuela","Vietnam","Virgin Islands (U.S.)","Yemen","Zambia","Zimbabwe"];
 
-  function getRandomIndex(items) {
+// API
+const api= "https://restcountries.eu/rest/v2/all"
+async function getData(){
+    const res= await fetch(api)
+    const data= await res.json()
+    var print= data.map(country => country.flag)

```

Comment:
```suggestion
    var print= data.map(country => country.flag)

```

In most cases, you should not use `var`. Check [this](https://blog.usejournal.com/awesome-javascript-no-more-var-working-title-999428999994) out for the reason.

Reply:
got it, thanks